src/features/perps/components/PerpsNavigatorFooter.tsx

```diff
@@ -217,7 +217,7 @@ const PerpsNewPositionScreenFooter = memo(function PerpsNewPositionScreenFooter(
     try {
       const livePrice = useLiveTokensStore.getState().tokens[getHyperliquidTokenId(market.symbol)].midPrice;
       const entryPrice = livePrice ?? market.price;
-      await hyperliquidAccountActions.createIsolatedMarginPosition({
+      const newPosition = await hyperliquidAccountActions.createIsolatedMarginPosition({
```

Comment:
Is it safe to assume no newPosition means cancellation from user? There are a couple other places this can return undefined https://github.com/rainbow-me/rainbow/blob/36a6097fc021b493d890c74b5977ac8a959c14a5/src/features/perps/services/hyperliquid-exchange-client.ts#L77, but not sure if those can actually happen?